gamemodes/terrortown/gamemode/shared/sh_main.lua

```diff
@@ -135,6 +135,15 @@ function GM:TTTPlayerColor(model)
 	return COLOR_WHITE
 end
 
+local UpdateSprint = UpdateSprint
```

Comment:
Please put variable definitions to the top of the file

Reply:
I put the variable definition there, because it's consistent with your code just a few lines further down:
https://github.com/TTT-2/TTT2/blob/77c5dbcee7ecc620d3ab7af7073106b1911c55b6/gamemodes/terrortown/gamemode/shared/sh_main.lua#L138-L145
I'll change it

Reply:
Jeah thats a thing where we would need a good linter for :D but we should put variables at the top, so that they can be found easily. Thanks :+1: